core/chains/evm/gas/suggested_price_estimator.go

```diff
@@ -177,8 +188,46 @@ func (o *SuggestedPriceEstimator) GetLegacyGas(ctx context.Context, _ []byte, Ga
 	return
 }
 
-func (o *SuggestedPriceEstimator) BumpLegacyGas(_ context.Context, _ *assets.Wei, _ uint32, _ *assets.Wei, _ []EvmPriorAttempt) (bumpedGasPrice *assets.Wei, chainSpecificGasLimit uint32, err error) {
-	return nil, 0, errors.New("bump gas is not supported for this chain")
+// Refresh the gas price in case the current one has gone stale.
+// The only reason bumping logic would be called on the SuggestedPriceEstimator is if there was a significant price spike
+// between the last price update and when the tx was submitted. Refreshing the price helps ensure the latest market changes are accounted for.
+func (o *SuggestedPriceEstimator) BumpLegacyGas(ctx context.Context, originalFee *assets.Wei, feeLimit uint32, maxGasPriceWei *assets.Wei, _ []EvmPriorAttempt) (newGasPrice *assets.Wei, chainSpecificGasLimit uint32, err error) {
+	chainSpecificGasLimit = feeLimit
+	ok := o.IfStarted(func() {
+		// Immediately return error if original fee is greater than or equal to the max gas price
+		// Prevents a loop of resubmitting the attempt with the max gas price
+		if originalFee.Cmp(maxGasPriceWei) >= 0 {
+			err = fmt.Errorf("original fee (%s) greater than or equal to max gas price (%s) so cannot be bumped further", originalFee.String(), maxGasPriceWei.String())
+			return
+		}
+		err = o.forceRefresh(ctx)
+		if newGasPrice = o.getGasPrice(); newGasPrice == nil {
+			err = errors.New("failed to refresh and return gas; gas price not set")
+			return
+		}
+		o.logger.Debugw("GasPrice", newGasPrice, "GasLimit", feeLimit)
+	})
+	if !ok {
+		return nil, 0, errors.New("estimator is not started")
+	} else if err != nil {
+		return
+	}
+	if newGasPrice != nil && newGasPrice.Cmp(maxGasPriceWei) > 0 {
+		return nil, 0, errors.Errorf("estimated gas price: %s is greater than the maximum gas price configured: %s", newGasPrice.String(), maxGasPriceWei.String())
+	}
+	// Add a buffer on top of the gas price returned by the RPC.
+	// Bump logic when using the suggested gas price from an RPC is realistically only needed when there is increased volatility in gas price.
+	// This buffer is a precaution to increase the chance of getting this tx on chain
+	bufferedPrice := fee.MaxBumpedFee(newGasPrice.ToInt(), o.cfg.BumpPercent(), o.cfg.BumpMin().ToInt())
```

Comment:
This is a weird thing to do here. Suggested estimator is supposed to only fetch what the RPC suggests. Although bumping seems good on paper it comes with a few drawbacks. On specific chains, the buffer needs to be disabled as it will mess up the tx inclusion unless you post exactly what the RPC suggests (i.e. Fantom, Klaytn). For chains that do want to enable this, we just rely on the same configs from the block history estimator, which might not be entirely applicable here. SCALE would have to figure those values out for existing and new chains.
Side note: if we keep the buffer feature, we would have to update `BumpPercent` description in the docs.

Reply:
Would it be possible to disable this buffering behavior by setting `BumpThreshold = 0`? 
I think most chains (except for Fantom) that use Suggested estimator has this set in their config, for example ([Klaytn](https://github.com/smartcontractkit/chainlink/blob/41f2497830959a79513d3e80bd5195bdef8b950b/core/chains/evm/config/toml/defaults/Klaytn_Mainnet.toml#L10), [Metis](https://github.com/smartcontractkit/chainlink/blob/41f2497830959a79513d3e80bd5195bdef8b950b/core/chains/evm/config/toml/defaults/Metis_Mainnet.toml#L15)). 
So we can make sure that the Suggested behavior is the same by setting the  `BumpThreshold ` as needed.
We can set this for Fantom to ensure backward compatibility.

+1 on updating the docs as well.

Reply:
Setting `BumpThreshold = 0` should disable bumping all together in the Confirmer so don't think this should be an issue for `Klaytn` or `Metis`.
I've updated the docs for `BumpPercent` but worried that it requires too much knowledge of our gas estimator implementation to understand. Let me know if you have had something else in mind. I also found references to a couple related configs `GasBumpPercent` and `GasBumpWei` that seem legacy. I replaced them with `BumpPercent` and `BumpMin` to clean them up.

Reply:
By default for SuggestedPrice, we already have BumpThreshold = 0
So bumping is OFF by default.
Whoever goes and changes that to enabled, must also understand that the bumpPercentrage and BumpMin will also apply to this estimator's bumping logic.

I think this is reasonable

Reply:
> By default for SuggestedPrice, we already have BumpThreshold = 0

@prashantkumar1982 where is this logic set? Looking at the config docs the `BumpThreshold` default seems to be 3 so looks like its not off by default for FTM .

https://github.com/smartcontractkit/chainlink/blob/develop/docs/CONFIG.md#bumpthreshold